lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -655,6 +657,32 @@ def test_html_view_for_invalid_certificate(self):
         self.assertIn("Cannot Find Certificate", response.content)
         self.assertIn("We cannot find a certificate with this URL or ID number.", response.content)
 
+    @override_settings(FEATURES=FEATURES_WITH_CERTS_ENABLED)
+    def test_html_lang_attribute_is_dynamic_for_invalid_certificate_html_view(self):
+        """
+        Tests that Certificate HTML Web View's lang attribute is based on user language.
+        """
+        self._add_course_certificates(count=1, signatory_count=2)
+        test_url = get_certificate_url(
+            user_id=self.user.id,
+            course_id=unicode(self.course.id)
+        )
+
+        # invalidate certificate and verify that "Cannot Find Certificate" is returned
+        self.cert.invalidate()
+
+        user_language = 'fr'
+        translation.activate(user_language)
```

Comment:
Use `with translation.override` to ensure that the language is properly switched back between assertions and after the test finishes. See https://docs.djangoproject.com/en/2.0/ref/utils/#django.utils.translation.override